src/test/java/dk/cngroup/lentils/service/CypherServiceTest.java

```diff
@@ -45,7 +45,7 @@ public void addNewCypher() {
 
         assertNotNull(cypher);
     }
-
+/*
```

Comment:
Radsi bych to smazal. Protoze jinak to tam zustane navzdy :D Kdyz to budes potrebovat, de se to krasne a rychle najit v historii toho souboru diky Gitu :)